src/models/IExtension.ts

```diff
@@ -1,25 +1,26 @@
 import { ILanguage } from './ILanguage';
 
-interface IExtension {
-  icon: string;
-  extensions: string[];
-  format: FileFormat;
-  alternate?: boolean;
-  light?: boolean;
+export interface IExtension {
```

Comment:
I'm aware of the use of barrels but I chosed simplicity in front of multiple files management when developing this. They're grouped by functionality.

Anyway, I'm ok with changing this to barrels. We can use folders as a way to group different `interfaces`.